app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/subscription/check_your_details.scala.html

```diff
@@ -0,0 +1,77 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.api.data.Form
+
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.config.ViewConfig
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.routes
+@import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.BusinessPartnerRecord
+@import scala.util.Try
+@import views.html.helper.CSRF
+
+@this(
+ mainTemplate: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.main_template,
+ formWrapper: uk.gov.hmrc.play.views.html.helpers.FormWithCSRF,
+ pageHeading: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.page_heading,
+ cyaSection: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_section,
+ cyaRow: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_row,
+ cyaChange: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_change,
+ addressDisplay: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.address_display,
+ submitButton: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button
+)
+
+@(bpr: BusinessPartnerRecord)(implicit request:Request[_], messages:Messages, appConfig: ViewConfig)
+
+@title = @{messages("subscription.title")}
+@label = @{Html(s"""<span class="form-label-bold">${messages("subscription.confirmLabel")}</span>""")}
+@emailAddress = @{bpr.emailAddress.map(e => e)}
```

Comment:
This map isn't doing anything. 
```scala
bpr.emailAddress.map(e => e)
``` 
is equivalent to just 
```
bpr.emailAddress
```
You might not need this val here

Reply:
you are right - it's not used at all!

Reply:
removed in latest commit